FileIO/CsvInterface.cpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * @file CsvInterface.cpp
+ * @author Karsten Rink
+ * @date 2015-03-25
+ * @brief Implementation of the CsvInterface class.
+ *
+ * @copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "CsvInterface.h"
+
+#include "GeoLib/Point.h"
+
+namespace FileIO {
+
+int CsvInterface::readPointsFromCSV(std::string const& fname, char delim, 
+                                    std::vector<GeoLib::Point*> &points)
+{
+	std::ifstream in(fname.c_str());
+
+	if (!in.is_open()) {
+		ERR ("CsvInterface::readPointsFromCSV(): Could not open file %s.", fname.c_str());
+		return -1;
+	}
+
+	std::string line;
+	getline(in, line);
+
+	std::size_t line_count(0);
+	std::size_t error_count(0);
+	std::list<std::string>::const_iterator it;
+	while ( getline(in, line) )
+	{
+		line_count++;
+		std::list<std::string> const fields = BaseLib::splitString(line, delim);
+
+		if (fields.size() < 3)
+		{
+			ERR ("Line %d contains not enough rows of data. Skipping line...", line_count);
+			error_count++;
+			continue;
+		}
+		it = fields.begin();
+		std::array<double, 3> point;
+		point[0] = strtod(it->c_str(), 0);
+		point[1] = strtod((++it)->c_str(), 0);
+		point[2] = strtod((++it)->c_str(), 0);
+		points.push_back(new GeoLib::Point(point[0], point[1], point[2]));
```

Comment:
I think it is comfortable to have such a constructor. I added it in PR https://github.com/ufz/ogs/pull/634.